metricbeat/module/windows/perfmon/_meta/docs.asciidoc

```diff
@@ -9,8 +9,9 @@ to collect. The example below collects processor time and disk writes.
 With `format` you can set the output format for a specific counter. Possible values are
 `float` and `long`. If nothing is selected the default value is `float`.
 With `instance_name`, you can specify the name of the instance. Use this setting when:
-- You want to use an instance name that is different from the computed name. For example, `Total` instead of `_Total`.
-- You specify a counter that has no instance. For example, `\TCPIP Performance Diagnostics\IPv4 NBLs/sec indicated without prevalidation`.
+
+* You want to use an instance name that is different from the computed name. For example, `Total` instead of `_Total`.
+* You specify a counter that has no instance. For example, `\TCPIP Performance Diagnostics\IPv4 NBLs/sec indicated without prevalidation`.
 For wildcard queries this setting has no effect.
```

Comment:
You need to add an empty line before line 15, or line 15 becomes another sentence in the second bullet point. I don't think thing that's what you intend here, correct?